src/views/documentation/Documentation.tsx

```diff
@@ -25,6 +25,7 @@ import { PageMeta } from "../../components/PageMeta";
 import { useSetFocus } from "../../hooks/useSetFocus";
 import { ExitSoknad } from "../../components/exit-soknad/ExitSoknad";
 import { SoknadHeader } from "../../components/soknad-header/SoknadHeader";
+import { ReadMore as readMore } from "../../components/sanity/readmore/ReadMore";
```

Comment:
Igjen så ser jeg ikke helt grunnen til å caste navnet her? Hvis det bare er for å få shorthand syntax inn i PortableText componenten så vet jeg ikke om jeg så fan 🤷